lib/airbrake/resque/failure.rb

```diff
@@ -7,13 +7,20 @@ module Failure
     # @see https://github.com/resque/resque/wiki/Failure-Backends
     class Airbrake < Base
       def save
+        return if activejob_exception?
         params = payload.merge(
           component: 'resque',
           action: payload['class'].to_s
         )
 
         ::Airbrake.notify_sync(exception, params)
       end
+
+      # AirBrake gem already attaches error notifications to ActiveJob
```

Comment:
Nitpick: `Airbrake` instead of `AirBrake`.
Overall I reckon this description belongs to line 10 more, so I think the following text above line 10 will do the job:

```
Avoid duplicate errors when the error is raised inside an ActiveJob job.
```